src/app.js

```diff
@@ -436,6 +436,41 @@ var run = function () {
 
   var renderer = new Renderer(editor, executionContext.web3(), updateFiles, udapp, executionContext, formalVerification.event, compiler.event); // eslint-disable-line
 
+  var autoCompile = document.querySelector('#autoCompile').checked;
+
+  document.querySelector('#autoCompile').addEventListener('change', function () {
```

Comment:
We have jquery here, so you could do some live stuff, but it's of course also fine like that.